src/client/components/card/CardModal.tsx

```diff
@@ -151,7 +159,7 @@ const CardModal: React.FC<CardModalProps> = ({
                   fallbackSrc="/content/default_card.png"
                   alt={cardName(card)}
                 />
-                {card?.details?.image_flip && (
+                {getCardBackImage(card) && (
```

Comment:
Dum dum by me. Everywhere else was using functions which ended up using functions from cardutil which handles custom images then fallback to the details images.